src/util/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export const randomColor = () =>
+  'hsla('.concat((360 * Math.random()).toString(), ', ', '70%,', '80%,1)');
```

Comment:
Quick and lazy solution. The contrast between the generated colors is not the best, however it does not really matter now since we will only have 1 asset.

Will be good to come up with a better solution to make sure colors have good contrast between each other and with the bg. Maybe a list of predefined colors could be a good idea?